js/packages/web/src/components/AuctionCard/index.tsx

```diff
@@ -884,12 +884,12 @@ export const AuctionCard = ({
               {notEnoughFundsToBid && (
                 <Text className="danger" type="danger">
                   You do not have enough funds to fulfill the bid. Your current
-                  bidding power is {biddingPower} SOL.
+                  bidding power is {biddingPower}.
```

Comment:
We will have to do more than remove the text here. See line #357 in same file

Reply:
Agree,

was pushing for sake of pushing, hadn't had time to think about namelookup. That's on the todo list